### ISSUE TITLE
Mods are gone (possibly) after replacing my computer motherboard

### ISSUE THREAD

**Issue by GamingSpud** [ModOrganizer2/modorganizer#2101]
## The problem:
I recently changed my computer motherboard (and subsequently had to struggle with boot drives, reactivating windows, audio device settings, etc.) and when I went to play some Fallout 4 via MO2 the instance that it was on (probably Skyrim SE or Fallout NV) had cero MO2 managed mods, and only showed DLCs and some mods I had downloaded before I found out about MO2. Upon reviewing my 5 instances, I found that both my SSE instances, as well as my FO:NV were missing al MO2 mods, and my FO3 instance only had 3 MO2 mods. Fortunately my FO4 instance seems to be intact, seeing as how I've been able to continue my "previously on hiatus" modded save without any crashes and was even able to mod it further (nothing major, just "ConsoleUtil", "Address Library" and "FOV Slider and Player Height").

## To Reproduce:

1. (I suppose) Change the motherboard of a computer with MO2 installed on the boot drive (outside the Program Files folder). (Was using an Asus TUF B350M PLUS Gaming, now I'm running an MSI B550M PRO-VDH WIFI, in case it matters)
2. After dealing with any boot, OS and driver issues that may have cropped up, open MO2 and check every instance to see if mods are still there.

## Environment:

* Mod Organizer Version that exhibits the issue: 2.4.4
* Last Mod Organizer Version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer: Windows 10

## Details:
![image](https://github.com/user-attachments/assets/059092b1-cf47-4e1d-8f79-bd5cd1725823)
This is the state of my NV instance, as you can see there are no MO2 mods in it, the Downloads tab is also completely empty.

![image](https://github.com/user-attachments/assets/d06949ad-9daa-4d05-8022-0e38200b63f7)
This is my FO3 instance, I had a lot more than just 3 MO2 mods, unsure as to why these 3 were spared the "purge".

### USVFS:
[https://gist.github.com/GamingSpud/8a2ab4b520e901a907d3ebf3781898d5](url)

### MO Interface:
[https://gist.github.com/GamingSpud/311a58511a704d7e0b11afb43580c1f8](url)

## Request:

Beyond just figuring out if my mobo swap was indeed the cause of the disappearance of my mods or how to fix them, I'd just like to ask if it's possible to find a record of the mods I was using and any possible setup configurations they had, so I can go in and "manually restore" the mods themselves.

**Comment by UhuruNUru** [comment]
This has nothing to do with MO2 & it's most likely a Windows drive mapping issue. it appears your MO" install can't find your mods folder.

Assuming you still have all your mods in your MO "Mods" folder on same drive, if  they are not showing up in MO2, 1st check MO2 is still looking in the right place, especially if done after reinstalling Windows, because if you do Windows will usually NEVER map your Drives to the same letters, I have never NOT reinstalled windows on a Mobo swap, so don't know for certain that it affects it, but as your mobo rmaps drives originally as you install new ones, a total swap of mobo can often result in Drive letters changing & old paths get broken.
Due to my setup, of reinstalling Windows yearly, I see this issue

Easiest way to check & fix this is use Disk Manager, to juggle Drive letters of the drives, (Or redo every path on your PC LOL) until they match your old Drive layout, I swap them out & back even if they appear correct, as a faisafe.

If you can't remember all the drive letters for your partitions, then checking saved paths Programs like MO2 expect, is your guide.

Hope that helps.

**Comment by GamingSpud** [comment]
Thank you! But there are a couple of things that don't pertain to my situation.

I actually didn't reinstall windows, just moved my components to a new mobo but I'm booting from the same drive I was before. Also, my drives are actually assigned to their letters (Boot drive is my M.2, assigned to C:/, extra storage SSD is assigned to F:/), and I went in and checked the mods folder of each instance, which were all empty except for the last 3 mods of my FO3 instance, as seen below:

![image](https://github.com/user-attachments/assets/4ae9707d-7096-4303-afc9-e200950106f8)

Was initially expecting the downloads tab of each instance to hold records of each download so I could fiddle with which mods I had active and their order, but that's gone too.

**Comment by Silarn** [comment]
By default MO2 stores files either in the MO2 install directory (if using a portable instance) or in AppData (if using a global instance). I would generally recommend moving the base directory for your instances to another location as AppData can get wiped and mangled by Windows for any number of reasons.

I'm not there so I can't tell you exactly why this happened but clearly the files were either moved or deleted. Or they're on a different drive path than MO2 is expecting, as @UhuruNUru speculated.

I can't really provide any more feedback than that.

**Comment by GamingSpud** [comment]
Just wanted to leave an update, after installing yet another M.2 and leaving it as "C:", the old M.2 as "D:" and the older SSD as "E:", I managed to find and restore my old MO2 Skyrim instance folder (which was not in "AppData" but in the "Downloads" folder of my old M.2).

Yes, I had to remap the instance base directory folders, I also had to move stuff from the "Documents/My Games/Skyrim Special Edition" of one drive to the other (And subsequently found that my computer had the "C:" drive "Documents" folder inside the ever annoying OneDrive folder...).

At the end of the day I managed to load my almost 4 year old save with no missing mods and no problems from what I can see. I still haven't found out why a change in motherboard and nothing but the motherboard redirected my instances to be in a different folder than they previously were (though if I had kept the default folder path when creating the instance in the first place this probably wouldn't have been a problem). If anyone else comes across this github issue after experiencing something similar that's the rundown of what happened to me.
